svelte.config.js

```diff
@@ -0,0 +1,28 @@
+// autogenerated and mostly unchanged,
+// except for (svelte-)inspector config
+
+import adapter from "@sveltejs/adapter-auto";
+import { vitePreprocess } from "@sveltejs/vite-plugin-svelte";
+
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+    extensions: [".svelte"],
+    // Consult https://kit.svelte.dev/docs/integrations#preprocessors
+    // for more information about preprocessors
+    preprocess: [vitePreprocess()],
+
+    vitePlugin: {
+        // inspect svelte components in running app
+        inspector: {
+            toggleKeyCombo: "control-alt-shift-i",
```

Comment:
Die Tastenkombination ist in Firefox eine standard Tastenkombination. Deswegen würde ich dies Vorschlagen
```suggestion
            toggleKeyCombo: "control-alt-i",
```